portfolio/src/main/java/com/google/sps/utils/ObjectDataStore.java

```diff
@@ -9,13 +9,21 @@
     /**
      * Puts the object into Datastore.
      * @param item Object needed to be stored.
+     * @return Id of the entity.
      */
-    public void store(T item);
+    public long store(T item);
 
     /**
      * Loads the object from Datastore.
      * @param limit Upper limit of number of objects.
      * @return An immutable list of objects.
      */
     public ImmutableList<T> load(int limit);
+
+    /**
+     * Deletes a specific object in Datastore.
+     * @param kind Entity kind name.
+     * @param id Target entity id.
+     */
+    public void delete(String kind, long id);
```

Comment:
Since each data store only handle one type, maybe we don't need to pass "kind"?

Reply:
Fixed